File: src/Workflow.Web.Service/Workflow/app/model/ticket/TicketActivity.js
Ext.define('Workflow.model.ticket.TicketActivity', {
    extend: 'Workflow.model.Base',
    fields: [
        { name: 'id', type: 'int' },
        { name: 'activityType', type: 'string' },
        { name: 'action', type: 'string' },
        { name: 'actionBy', type: 'string' },
        { name: 'description', type: 'string' },
        { name: 'createdDate', type: 'date' },
        { name: 'modifiedDate', type: 'date' },
        { name: 'version', type: 'int' }
    ]
});

File: src/Workflow.Domain/Ticket/TicketActivity.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using Workflow.Domain.Entities.Base;

namespace Workflow.Domain.Entities.Ticket
{
    public class TicketActivity : AbstractEntityLevel2
    {
        public int TicketId { get; set; }
        public string ActivityType { get; set; }
        public string Action { get; set; }
        public int ActionBy { get; set; }
        public Ticket Ticket { get; set; }

    }
}
